src/turbolinks/controller.coffee

```diff
@@ -217,6 +218,9 @@ class Turbolinks.Controller
     root = @getSetting("root") ? "/"
     new Turbolinks.Location root
 
+  isCacheable: ->
```

Comment:
I think `shouldCacheSnapshot` might be a clearer name. I'd also move this method to the "Snapshot cache" section above, just after `cacheSnapshot`.